test/with_top_algebra.lean

```diff
@@ -0,0 +1,15 @@
+import algebra.ordered_ring
```

Comment:
I think we have a file called "instance diamonds" this could go in

Reply:
https://github.com/leanprover-community/mathlib/blob/master/test/instance_diamonds.lean